project1.css

```diff
@@ -29,17 +31,45 @@ section {
   top: 92px;
 }
 
+@media screen and (min-width: 480px) {
+  section {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 114px 24px;
+    gap: 12px;
+    position: absolute;
+    width: 1400px;
+    height: 916px;
+    left: calc(30% - 375px / 2);
+    top: 142px;
+  }
+}
+
 .background-image {
   background-image: url("header-shapes mobile@2x.png");
   background-size: contain;
   background-repeat: no-repeat;
   position: absolute;
   width: 375px;
   height: 846px;
-  left: 0;
+  left: 20;
   top: 40px;
 }
 
+@media screen and (min-width: 480px) {
```

Comment:
- [ ] I would recommend you increase the breakpoint of your media query to 768px or more for the desktop view. 480px is too small and it falls into the category of the mobile version. Kindly adjust it 😄 .